### DISCUSSION TITLE
Idea Discussion: Enhancing Oqtane.Docs Homepage

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.docs#28] [Ideas]
# Enhancing Oqtane.Docs Homepage

I have an idea to enhance the Oqtane.Docs homepage by adding more project-related information for users and contributors. This could include details such as the documentation version, roadmap, acknowledgments, contribution guidelines, and improved getting started instructions.

## Proposed Changes
- Display current documentation version and last update date.
- Add a roadmap section outlining upcoming releases and features.
- Acknowledge and appreciate contributors.
- Provide clear guidelines on how to contribute.
- [Improve the "Getting Started"](https://github.com/oqtane/oqtane.docs/issues/26) section with concise instructions.

## Additional Context
The goal is to create a more informative and inviting homepage, making it easier for both new users and contributors to understand the project's current state and how they can get involved.

## Open for Discussion
This post is open for discussion and feedback from the community. What are your thoughts on these proposed changes? Do you have additional ideas or suggestions? Let's brainstorm and make the Oqtane.Docs homepage even better together!

Reference: [Oqtane.Framework Project Home Page Read Me](https://github.com/oqtane/oqtane.framework)

**Comment by iJungleboy** [comment]
Fully agree ;)

**Comment by thabaum** [comment]
@iJungleboy Looks like you are hard at work on it making this Oqtane.Docs ride smooth out for the autobahn! Thank you!
